library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -440,4 +440,8 @@ SAMLGeneral_b=An error occurred while validating the signature. Make sure that a
 
 SAMLGeneral_c=This project is intended to be a set of blackbox tests that verify the conformance of \
   an IdP/SP to the SAML V2.0 Standard Specification. It only supports SAML Version 2.0 and all other \
-  version are not supported.
\ No newline at end of file
+  version are not supported.
+
+SAMLGeneral_d=An error test received a non-error HTTP status code. \n \
+  Note that if the IdP sends SAML error responses with a <StatusCode>, use the -e flag when \
```

Comment:
We **need** both `-l` and `-e` together to run error tests and be lenient. 

I actually wanted to propose getting rid of `-e`. Initially, the motivation for adding the `-e` was because IdPs may not respond correctly and throw exceptions but now that we have `-l`. We can always run the negative path tests. Plus if these negative tests fail with the `-l` option, it mean the IdP is doing something comply wrong and signing people in when it shouldn't.